random-beans/src/main/java/io/github/benas/randombeans/randomizers/misc/EnumRandomizer.java

```diff
@@ -32,7 +32,7 @@
  *
  * @author Mahmoud Ben Hassine (mahmoud.benhassine@icloud.com)
  */
-public class EnumRandomizer<T extends Enum<T>> extends AbstractRandomizer<Enum<T>> {
+public class EnumRandomizer<T> extends AbstractRandomizer<T> {
```

Comment:
Hi,

I'm not sure I understand the purpose of removing this bound. This will allow to have `EnumRandomizer<Integer> ir = new EnumRandomizer<>(Integer.class);` which makes no sense. The idea was to restrict generic types to only enums for `EnumRandomizer`. You can still write `EnumRandomizer<Month> mr = new EnumRandomizer<>(Month.class);` even with the bounded generic type since `Month` is an enum. Do you agree?

Cheers
Mahmoud